README.md

```diff
@@ -1,10 +1,72 @@
 # Democrafy cli
+This is one of the user applications of the Democrafy System. The Democrafy-cli lets you interact with the Democrafy server by requesting data 
+of the current song and requesting that the song is skipped.This is the only application you have to 
+download being a normal user. 
```

Comment:
If we are going to play with the democracy jargon, I would say `...download being a regular citizen`

Reply:
And it's missing the clarification that you need it only if you want to use the CLI, but it's not necessary for experiencing **Democrify®**

Reply:
> This is the only application you have to 
download being a normal user. 

Esta es la parte que dice que solo hace falta para el cli. Esta primera parte habla solo del Democrafy-cli. Ya cambie lo de citizen.

Reply:
Al final me parecio muy confusa la frase, solo deje que esssssss la unica aplicacion que hace falta descargar